indico/modules/events/registration/__init__.py

```diff
@@ -94,6 +98,16 @@ def _get_management_roles(sender, **kwargs):
     return RegistrationRole
 
 
+@signals.get_placeholders.connect_via('registration-email')
+def _get_registration_placeholders(sender, registration, **kwargs):
+    yield FirstNamePlaceholder
+    yield LastNamePlaceholder
+    yield EventTitlePlaceholder
+    yield EventURLPlaceholder
```

Comment:
EventLinkPlaceholder, and returning raw URLs is a bit problematic since there is no easy way to actually turn them into clickable links in CKEditor. Putting `{whatever}` in the URL field in the link dialog won't work since it prepends a protocol.

For agreements and invitations I simply have a placeholder that returns a piece of HTML with both the link target and the link text being the URL. I'd go for the same thing here. It's not super pretty since someone might prefer a different title, but that's life. Or we could have another one with the event title as the link title...